<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rc 转 ui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/tiny_refresh_plugins.png</normaloff>:/images/tiny_refresh_plugins.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout1">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>源码位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="txtSourcePath">
          <property name="text">
           <string>F:/AppCode/notepad-plus-plus-master/PowerEditor/src</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSourcePath">
          <property name="text">
           <string>选择(&amp;B)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainLayout2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输出位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="txtTargetPath">
          <property name="text">
           <string>C:/Users/nanping/Desktop/666</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTargetPath">
          <property name="text">
           <string>选择(&amp;T)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="txtLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
  </widget>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/debug_continue.png</normaloff>:/images/debug_continue.png</iconset>
   </property>
   <property name="text">
    <string>执行(&amp;R)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
